monai/networks/nets/autoencoderkl.py

```diff
@@ -622,7 +520,6 @@ class AutoencoderKL(nn.Module):
         norm_eps: epsilon for the normalization.
         with_encoder_nonlocal_attn: if True use non-local attention block in the encoder.
         with_decoder_nonlocal_attn: if True use non-local attention block in the decoder.
-        use_flash_attention: if True, use flash attention for a memory efficient attention mechanism.
```

Comment:
May I know remove this will not affect anything here, right?

Reply:
`flash_attention` was just a faster version of attention but exactly the same results. the implementation here  never worked that well, we never used it, and it will make sense to use pytorch's inbuilt version in future so i think it is fine to remove. because it was never used i don't think it will affect anything if we remove it

Reply:
Got it.